src/core_landice/Registry.xml

```diff
@@ -172,6 +180,46 @@
+                        description="If true, then turn on the li_finalize_damage_after_advection function."
+		            possible_values=".true. or .false."
+                />
+		<nml_option name="config_preserve_damage" type="logical" default_value=".false." units="unitless"
+                        description="If true, then we preserve the damage before advection (no heal)"
+		            possible_values=".true. or .false."
+                />
+		<nml_option name="config_calculate_damage" type="logical" default_value=".false." units="unitless"
+                        description="If true, then turn on the calculate_damage function."
+		            possible_values=".true. or .false."
+                />
+                <nml_option name="config_damage_preserve_threshold" type="real" default_value="0.0" units="unitless"
+                        description="It is a value above which the old damage number would be kept from the impact of advection (the damage would not heal). For example, if the damage at some location is 0.83, and we set config_damage_preserve_threshold to 0.8, then the damage number there will not fall below 0.83. Otherwise, if the damage is 0.7, then it could be decreased to a smaller number if non-damaged ice is transported from upstream. If we set config_damage_preserve_threshold to 0 (default), then it will just not play a role in pratical."
+	                possible_values="real value between 0 and 1"
+		/>
+                <nml_option name="config_damage_calving_threshold" type="real" default_value="1.0" units="unitless"
+                        description="It is a value above which the cell would be calved away."
+	                possible_values="real value between 0 and 1"
+		/>
+                <nml_option name="config_damage_stiffness_min" type="real" default_value="0.1" units="unitless"
+                        description="The minimum value the stiffnessFactor can take due to evolving damage when stiffnessFactor is calculated from damage (config_damage_rheology_coupling=.true.). Used to prevent unrealistic and/or numerically challenging stiffness values. For example, if damage = 0.9, then stiffnessFactor is 1 - damage = 0.1. But if we set config_damage_stiffness_min = 0.5, then stiffnessFactor can only be allowd as 0.5"
+	                possible_values="real value between 0 and 1"
+		/>
+                <nml_option name="config_damage_rheology_coupling" type="logical" default_value=".false." units="unitless"
+                        description="If true, then we change stiffnessFactor to 1-damage during the run."
+	                possible_values=".true. or .false."
+		/>
+		        <nml_option name="config_damage_gl_setting" type="character" default_value="none" units="unitless"
+		                description="Selection of the method for damage setting near the grounding line"
+		            possible_values="'nye', 'extra'"
```

Comment:
It might also be helpful here to clarify what happens if "none" (the default) is chosen. If the code won't function sensibly with this set to 'none' (e.g., existing damage would eventually be swept out of the domain but new damage would not be initialized) then maybe 'none' should not be a viable option? Or, if 'none' is only useful for testing / debugging then as Matt suggests, 'nye' should be the default?

Reply:
Ok. We can change 'extra' to 'extrapolate' and make 'nye' as default. I don't have strong preference with it. The 'none' option is not physical, as we always want to let the damage near GL stay above some value instead of being healed quickly by the grounded and undamaged ice from upstream.